index.html

```diff
@@ -1,180 +1,180 @@
-        <div class="btn">
-            <a href="#" class="resume">Get my resume</a>
-        </div>
-        <div class="skill_list">
-            <div class="title1">
-                <h2>Languages</h2>
-                <a href="#" class= "fa fa-angle-down"></a>
-                <ul class="langlist">
-                <li class="list1">
-                    <img src="/image/HTML.png" alt="HTML icon">
-                    <p>JavaScript</p>
-                </li>
-                <li class="list2">
-                    <img src="/image/CSS.png" alt=" CSS icon">
-                    <p>HTML</p>
-                </li>
-                <li class="list3">
-                    <img src="/image/JS.png" alt="JS icon">
-                    <p>CSS</p>
-                </li>
-                </ul>
-            </div>
-            <div class="title2">
-                <h2>Frameworks</h2>
-                <a href="#" class= "fa fa-angle-right"></a>
-            </div>
-            <div class="title3">
-                <h2>Skills</h2>
-                <a href="#" class= "fa fa-angle-right"></a>
-            </div>
-        </div>
-        </div>
-    </div>
-
-    <!-- Contact me
-    <div class="contact">
-        <h1>Contact me</h1>
-        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
-            I’d love to help with it! </p>
-        <input type="text" id="fname" name="fname" value="Lorem Ipsum">
-        <input type="text" id="email" name="email" value="Loremipsum@gmail.com">
-        <input type="text" id="msg" name="msg" value="Write your message here...">
-        <div class="submit">
-            <a href="#" class="tt">Get in touch</a>
-        </div>
-    </div> -->
-    </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <a href="#">My Logo</a>
+        <nav> 
+        <a href="#" class="fa fa-bars"></a>
+        </nav>
+    </header>
+
+    <section class="hero">
+        <h1>I'm Abhishek. <br> Glad to see <br> you !</h1>
+        <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me. </p>
+        <a href="#">LET'S CONNECT</a>
+            <ul class="social">
+                <li><img src="/image/Vector.svg"></li>
+                <li><img src="/image/Linkedin_icon.svg"></li>
+                <li><img src="/image/Vector1.svg"></li>
+                <li><img src="/image/Vector2.svg"></li>

```

Comment:
- [ OPTIONAL ] Your images are not showing, I have modified the HTML to add a `.` before the /, and there we go, I think you need to review this part to implement the changes, see?
<img width="1036" alt="1" src="https://github.com/Nomarg/My-Portfolio/assets/77843050/34351f4e-f7de-4929-ae28-717391e26d62">